src/components/buttons/EdgeSwitch.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState } from 'react'
+import { TouchableOpacity, View } from 'react-native'
+
+import { cacheStyles, Theme, useTheme } from '../services/ThemeContext'
+import { EdgeText } from '../themed/EdgeText'
+
+interface EdgeSwitchProps {
+  labelA: string
+  labelB: string
+  onSelectA: () => void
+  onSelectB: () => void
+}
+
+const EdgeSwitch: React.FC<EdgeSwitchProps> = ({ labelA, labelB, onSelectA, onSelectB }) => {
+  const [isASelected, setIsASelected] = useState(true)
+  const theme = useTheme()
+  const styles = getStyles(theme)
+
+  const handlePress = () => {
+    if (isASelected) {
+      onSelectB()
+    } else {
+      onSelectA()
+    }
+    setIsASelected(!isASelected)
+  }
+
+  return (
+    <View style={styles.container}>
+      <TouchableOpacity onPress={handlePress} style={styles.touchContainer}>
+        <View style={[styles.option, isASelected && styles.selectedOption]}>
+          <EdgeText style={styles.label}>{labelA}</EdgeText>
+        </View>
+        <View style={[styles.option, !isASelected && styles.selectedOption]}>
+          <EdgeText style={styles.label}>{labelB}</EdgeText>
+        </View>
+      </TouchableOpacity>
+    </View>
+  )
+}
+
+const getStyles = cacheStyles((theme: Theme) => ({
+  container: {
+    flexDirection: 'row',
+    justifyContent: 'center',
+    margin: theme.rem(0.5)
+  },
+  touchContainer: {
+    flexDirection: 'row',
+    alignSelf: 'flex-start',
+    borderRadius: theme.rem(1.5),
+    backgroundColor: theme.cardBaseColor,
+    overflow: 'hidden'
+  },
+  selectedOption: {
+    backgroundColor: theme.cardBaseColor,
+    borderRadius: theme.rem(1.5)
+  },
+  option: {
+    paddingVertical: 8,
+    paddingHorizontal: theme.rem(2),
+    alignItems: 'center',
+    justifyContent: 'center'
+  },
+  label: {}
+}))
+
+export default EdgeSwitch
```

Comment:
Never use `export default`! It is 100% banned in all Edge projects. We used named exports for everything, so do `export const EdgeSwitch` avove instead.

Reply:
Whoops, missed AI copy pasta mistake, sorry. I'll try to fix my prompt

Reply:
Came from an AI prompt change, my bad

Reply:
Yes, enabling `no-default-export` would be a welcome addition.